sigstore-java/src/test/java/dev/sigstore/tuf/FileSystemTufStoreTest.java

```diff
@@ -45,7 +45,7 @@ void setTrustedRoot_noPrevious(@TempDir Path repoBase) throws IOException {
     TufLocalStore tufLocalStore = FileSystemTufStore.newFileSystemStore(repoBase);
     assertFalse(repoBase.resolve("root.json").toFile().exists());
     tufLocalStore.storeTrustedRoot(TestResources.loadRoot(TestResources.UPDATER_REAL_TRUSTED_ROOT));
-    assertEquals(1, repoBase.toFile().list().length);
+    assertEquals(2, repoBase.toFile().list().length); // root.json plus the targets dir.
```

Comment:
Could you move code comment to the assertion message?

Reply:
done.